README.md

```diff
@@ -198,6 +198,17 @@ You can use the simpleTest.py script to run various configurations via a simple
 command line interface.
 Please find more information [here](./tests/simpleTest/README.md)
 
+### Example application
+This example application shows the full capabilities of the Concord-BFT consensus-based byzantine fault-tolerant state machine replication library.
```

Comment:
Maybe not the full capabilities, because the demo doesn't include things like reconfiguration  right?

Reply:
Thanks @teoparvanov I will update the readme